sceptre/config/reader.py

```diff
@@ -429,49 +446,64 @@ def _render(self, directory_path, basename, stack_group_config):
         """
         config = {}
         abs_directory_path = path.join(self.full_config_path, directory_path)
-        if path.isfile(path.join(abs_directory_path, basename)):
-            default_j2_environment_config = {
-                "autoescape": select_autoescape(
-                    disabled_extensions=("yaml",),
-                    default=True,
-                ),
-                "loader": FileSystemLoader(abs_directory_path),
-                "undefined": StrictUndefined,
-            }
-            j2_environment_config = strategies.dict_merge(
-                default_j2_environment_config,
-                stack_group_config.get("j2_environment", {}),
+
+        if not path.isfile(path.join(abs_directory_path, basename)):
+            return
```

Comment:
I don't understand what the point of this if statement is or whether this is reachable but it seems clearer to rewrite it this way than have the main logic inside an if branch.